metrics.go

```diff
@@ -59,23 +59,26 @@ func (m *metrics) Request() {
 	}
 }
 
-func (m *metrics) Success() {
+func (m *metrics) Success(t time.Time) {
 	atomic.AddInt32(&m.g.Stats.Successful, 1)
 	if m.statsd.c != nil {
+		m.statsd.c.Timing("latency.success", time.Since(t).Milliseconds())
```

Comment:
confirming `Timing` metric gets submitted to statsd lib in milliseconds.
https://github.com/alexcesaro/statsd/blob/v2.0.0/statsd.go#L108